vocabularies/qld-mining-permits.ttl

```diff
@@ -0,0 +1,305 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix minpermit: <http://linked.data.gov.au/def/qld-mining-permit/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/qld-mining-permit> a owl:Ontology .
+
+minpermit:conceptScheme a skos:ConceptScheme ;
+    dct:created "2019-03-29"^^xsd:dateTime ;
+    dct:modified "2019-09-23"^^xsd:dateTime ;
+    skos:altLabel "Resource Authority"@en,
+        "Tenure"@en ;
+    skos:definition """In general mineral or energy rights in Australia are reserved to the Crown. The acquisition of rights to minerals or energy stems from legislative frameworks in Queensland. These frameworks provide initially for exploration of the resource, and consist of the grant by the Crown in the form of exploration permits, licences or leases.
```

Comment:
triple quotation marks at start and end of this definition

Reply:
https://www.w3.org/TR/sparql11-query/#QSynLiterals

To facilitate writing literal values which themselves contain quotation marks or which are long and contain newline characters, SPARQL provides an additional quoting construct in which literals are enclosed in three single- or double-quotation marks.

The lines affected contain the newline character \n

Reply:
This is a valid skos syntax. Triple quotation marks are use where there are characters that otherwise would require escaping (e.g. \n). Using triple quotation marks removes the need to escape characters.